scripts/deploy.py

```diff
@@ -362,19 +336,135 @@ def _execute_deployment():
                 'file a blocking bug and switch to the last known good '
                 'version.')
 
-    python_utils.PRINT('Done!')
-
 
-def get_unique_id():
-    """Returns a unique id.
+def execute_deployment():
+    """Executes the deployment process after doing the prerequisite checks.
 
-    Returns:
-        str. The unique id to be returned.
+    Raises:
+        Exception: The deployment script is not run from a release branch.
+        Exception: Current release version has '.' character.
+        Exception: The mailgun API key is not added before deployment.
+        Exception: Could not find third party directory.
+        Exception: Invalid directory accessed during deployment.
```

Comment:
You have missed a few exceptiond you have in the code.

Reply:
Updated.